dev/breeze/doc/ci/07_running_ci_locally.md

```diff
@@ -0,0 +1,129 @@
+|-------------------|-----------|------------|-----------------------------------------|
+| HOST_USER_ID      | Host UID  |            | User id of the host user.               |
+| HOST_GROUP_ID     | Host GID  |            | Group id of the host user.              |
+| HOST_OS           | <from os> | linux      | OS of the Host (darwin/linux/windows).  |
+| COMMIT_SHA        |           | GITHUB_SHA | SHA of the commit of the build is run   |
+
+## In-container environment initialization
+
+| Variable                        | Local dev | CI        | Comment                                                                     |
+|---------------------------------|-----------|-----------|-----------------------------------------------------------------------------|
+| SKIP_ENVIRONMENT_INITIALIZATION | false (*) | false (*) | Skip initialization of test environment (*) set to true in pre-commits.     |
+| SKIP_IMAGE_UPGRADE_CHECK        | false (*) | false (*) | Skip checking if image should be upgraded (*) set to true in pre-commits.   |
+| SKIP_PROVIDERS_TESTS            | false     | false     | Skip running provider integration tests.                                    |
+| SKIP_SSH_SETUP                  | false     | false (*) | Skip setting up SSH server for tests. (*) set to true in GitHub CodeSpaces. |
+| VERBOSE_COMMANDS                | false     | false     | Whether every command executed in docker should be printed.                 |
+
+# Image build variables
+
+| Variable                        | Local dev | CI        | Comment                                                            |
+|---------------------------------|-----------|-----------|--------------------------------------------------------------------|
+| UPGRADE_TO_NEWER_DEPENDENCIES   | false     | false (*) | Whether dependencies should be upgraded. (*) set in CI when needed |
+
+# Upgrade to newer dependencies
+
+By default, we are using a tested set of dependency constraints stored in separated "orphan" branches of the airflow repository
+("constraints-main, "constraints-2-0") but when this flag is set to anything but false (for example random value),
+they are not used and "eager" upgrade strategy is used when installing dependencies. We set it to true in case of direct
+pushes (merges) to main and scheduled builds so that the constraints are tested. In those builds, in case we determine
+that the tests pass we automatically push latest set of "tested" constraints to the repository. Setting the value to random
+value is best way to assure that constraints are upgraded even if there is no change to pyproject.toml
+This way our constraints are automatically tested and updated whenever new versions of libraries are released.
+(*) true in case of direct pushes and scheduled builds
+
+----
+
+**Thank you** for reading this far. We hope that you have learned a lot about Airflow's CI.
```

Comment:
I learned a lot :-D by reading until the very end!

Reply:
yes me too :)

Reply:
Cool :). ❤️